docs/reference-docs/rug/rug-conventions.md

```diff
@@ -28,22 +26,23 @@ concrete examples.
 
 [travis-editors]: https://github.com/atomist-rugs/travis-editors
 
-### Rug Archive
+### Rug Projects
```

Comment:
They are different things - and is a point of correctness. A project laid out on disc is _not_ an archive.